front/src/reducers/__tests__/userReducer.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, test } from 'vitest';
+import { createStoreWithoutMiddleware } from 'Store';
+import {
+  userInitialState,
+  loginSuccess,
+  loginError,
+  logoutUser,
+  serverError,
+  updateAccount,
+  UserState,
+} from 'reducers/user/userSlice';
+
+const createStore = (initialStateExtra?: UserState) =>
+  createStoreWithoutMiddleware({
+    user: initialStateExtra,
+  });
+
+describe('userReducer', () => {
+  test('should return the initial state', () => {
+    const store = createStore(userInitialState);
+    const userState = store.getState().user;
+    expect(userState).toEqual(userInitialState);
+  });
+
+  test('should handle loginSuccess', () => {
+    const store = createStore(userInitialState);
+    const action = loginSuccess({ accessToken: '', username: 'Test userSlice' });
```

Comment:
```suggestion
    const action = loginSuccess({ accessToken: 'fake_token', username: 'Test userSlice' });
```
Here use something that’s not going to be a default value, or something that can end up here by chance

Reply:
Thank you, done !!